src/main/webapp/scripts/pixelation.js

```diff
@@ -64,14 +63,14 @@ function animate_update(endpoint) {
 
         if (PIXEL_FACTOR_CURR < target) {
             PIXEL_FACTOR_CURR += dx;
-            // "Binary Search" appraoch to hone in on exact value since dx > 1
+            // "Binary Search" approach to home in on exact value since dx > 1
```

Comment:
I think this was actually was supposed to be 'hone'

Reply:
Yeah, but I think both work anyways